pypd/mixins.py

```diff
@@ -45,6 +46,13 @@ def __init__(self, api_key=None, base_url=None):
             from pypd import base_url
             self.base_url = base_url
 
+        if proxies:
```

Comment:
**Unit tests** - problem here is that I'm not sure how you'd test a client proxy config without a proxy already setup to tests against. I think we could achieve this mocking? What's your take on it?

**README** - I've fleshed out the proxy section a little bit. Let me know if it's still unclear.